<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulationWindow</class>
 <widget class="QWidget" name="SimulationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Simulation Time:</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdTimer">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>211</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="elevator1Floor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor: nil</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elevator: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator1Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status: Moving</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator1Doors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doors: Closed</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator1Passengers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Passengers: 1</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>211</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elevator: 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator2Floor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor: nil</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator2Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status: Moving</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator2Doors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doors: Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator2Passengers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Passengers: 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>211</width>
     <height>291</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elevator: 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator3Floor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor: nil</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator3Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Status: Idle</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator3Doors">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doors: Closed</string>
    </property>
   </widget>
   <widget class="QLabel" name="elevator3Passengers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Passengers: 5</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>271</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor1Up">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†‘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2Up">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†‘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor2Down">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†“</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3Up">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†‘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor3Down">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†“</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4Down">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†“</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor4Up">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†‘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floor 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Request Elevator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floor5Down">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>â†“</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>381</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Elevator Control Panel (Inside)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destFloor1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destFloor2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destFloor3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destFloor4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="destFloor5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDoor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCloseDoor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Door</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFire">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDoorObstacle">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Door Obstacle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPowerOutage">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Power Outage</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="eventLog">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>411</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
